components/tooltip/tooltip.directive.ts

```diff
@@ -33,6 +34,8 @@ export class TooltipDirective {
   private visible:boolean = false;
   private tooltip:ComponentRef<any>;
 
+  private delayTimeoutId:NodeJS.Timer;
```

Comment:
At first I used type number, but when I created the pull request, travis cancel the build because `Error: Error at /home/travis/build/valor-software/ng2-bootstrap/components/tooltip/tooltip.directive.ts:53:5: Type 'Timer' is not assignable to type 'number'.`